BDArmory/Weapons/Missiles/MultiMissileLauncher.cs

```diff
@@ -245,6 +244,12 @@ IEnumerator DelayedStart()
                             if (parts.Current.partPrefab.partInfo.name != subMunitionName) continue;
                             var explosivePart = parts.Current.partPrefab.FindModuleImplementing<BDExplosivePart>();
                             bRadius = explosivePart != null ? explosivePart.GetBlastRadius() : 0;
+                            var ML = parts.Current.partPrefab.FindModuleImplementing<MissileLauncher>();
+                            if (!string.IsNullOrEmpty(subMunitionName))
```

Comment:
This should probably check that `ML` isn't `null` and throw some sort of error if it is.